sauce/features/general/hide-help/index.js

```diff
@@ -0,0 +1,61 @@
+import { Feature } from 'core/feature';
+
+export class hideHelp extends Feature {
+
+  injectCSS() { return require('./index.css'); }
+
+  shouldInvoke() {
+    if (this.settings.enabled === true) {
+      return true;
+    }
+  }
+
+  observe(changedNodes) {
+    if (!this.shouldInvoke()) return;
```

Comment:
you won't need this since it will always return true, you can just remove the check